src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1403,31 +1403,23 @@ private BoundExpression BindSizeOf(SizeOfExpressionSyntax node, BindingDiagnosti
         }
 
         /// <returns>true if managed type-related errors were found, otherwise false.</returns>
-        internal static bool CheckManagedAddr(CSharpCompilation compilation, TypeSymbol type, bool warnForManaged, Location location, BindingDiagnosticBag diagnostics)
+        internal static bool CheckManagedAddr(CSharpCompilation compilation, TypeSymbol type, Location location, BindingDiagnosticBag diagnostics)
         {
             var useSiteInfo = new CompoundUseSiteInfo<AssemblySymbol>(diagnostics, compilation.Assembly);
             var managedKind = type.GetManagedKind(ref useSiteInfo);
             diagnostics.Add(location, useSiteInfo);
 
-            return CheckManagedAddr(compilation, type, managedKind, warnForManaged, location, diagnostics);
+            return CheckManagedAddr(compilation, type, managedKind, location, diagnostics);
         }
 
         /// <returns>true if managed type-related errors were found, otherwise false.</returns>
-        internal static bool CheckManagedAddr(CSharpCompilation compilation, TypeSymbol type, ManagedKind managedKind, bool warnForManaged, Location location, BindingDiagnosticBag diagnostics)
+        internal static bool CheckManagedAddr(CSharpCompilation compilation, TypeSymbol type, ManagedKind managedKind, Location location, BindingDiagnosticBag diagnostics)
         {
             switch (managedKind)
             {
                 case ManagedKind.Managed:
-                    if (warnForManaged)
-                    {
-                        diagnostics.Add(ErrorCode.WRN_ManagedAddr, location, type);
-                        return false;
-                    }
-                    else
-                    {
-                        diagnostics.Add(ErrorCode.ERR_ManagedAddr, location, type);
-                        return true;
-                    }
+                    diagnostics.Add(ErrorCode.WRN_ManagedAddr, location, type);
```

Comment:
We should also do a LangVersion check so that people on old language versions can't suppress these

Reply:
I think that's a good idea. I'll do a `runtime` rebuild tonight to verify they don't have a project with managed pointers and LangVer<11.

Reply:
Following conversation with Jared, I added a LangVer check. Thanks